MONAIAuto3DSeg/Scripts/auto3dseg_segresnet_inference.py

```diff
@@ -109,34 +91,17 @@ def main(model_file,
     model = model.to(device=device, memory_format=torch.channels_last_3d)  # gpu
     model.eval()
 
-    # make input Transform chain
-    main_normalize_mode = config["normalize_mode"]
-    intensity_bounds = config["intensity_bounds"]
-    if len(keys) == 1:  # only one input image
-        images_loaded['image'] = images_loaded['image1']
-        ts = [
-            EnsureTyped(keys="image", data_type="tensor", dtype=torch.float, allow_missing_keys=True)
-        ]
-        _add_normalization_transforms(ts, "image", main_normalize_mode, intensity_bounds)
-    else:  # multiple input images
-        ts = [
-        ]
-
-        extra_modalities = OrderedDict(config['extra_modalities'])
```

Comment:
Has this been removed intentionally? How do you know what normalization is needed for each modality?

Reply:
Yes, this was removed intentionally. I've tested on prostate and brats and it worked well. Just the case where an extra channel being a labelmap is not supported